shared/version/api.go

```diff
@@ -395,6 +395,7 @@ var APIExtensions = []string{
 	"oidc_groups_claim",
 	"loki_config_instance",
 	"storage_volatile_uuid",
+	"migration_stateful_default",
```

Comment:
`instances_migration_stateful` like `instances_nic_host_name` extension - please try to keep to conventions.